chef/transport.go

```diff
@@ -34,15 +35,19 @@ func MakeHandler(service Service, logger kitlog.Logger, ml machineLearning.Servi
 
 	getEnvironmentForGroup := kithttp.NewServer(gokitjwt.NewParser(gokit.GetJWTKeys(), jwt.SigningMethodHS256,
 		telegram.CustomClaimFactory)(makeGetEnvironmentForGroupEndpoint(service)), gokit.DecodeString, gokit.EncodeResponse, opts...)
-
+	getChefRecipesByGroup := kithttp.NewServer(makeGetChefRecipesByGroupEndpoint(service), handleRequest, gokit.EncodeResponse, opts...)
+	getChefEnvironmentsByGroup := kithttp.NewServer(makeGetChefEnvironmentsByGroupEndpoint(service), handleRequest, gokit.EncodeResponse, opts...)
+	getChefNodes := kithttp.NewServer(makeGetChefNodesEndpoint(service), handleGetNodesRequest, gokit.EncodeResponse, opts...)
 	r := mux.NewRouter()
 
 	r.Handle("/api/chef/delivery/{chatid:[0-9]+}", chefDeliveryEndpoint).Methods("POST")
 	r.Handle("/api/chef/recipe", addChefRecipeToGroup).Methods("POST")
 	r.Handle("/api/chef/recipes", getChefRecipesForGroup).Methods("GET")
 	r.Handle("/api/chef/environment", addEnvironmentToGroup).Methods("POST")
 	r.Handle("/api/chef/environments", getEnvironmentForGroup).Methods("GET")
-
+	r.Handle("/api/chef/environments/group/{groupid}", getChefEnvironmentsByGroup).Methods("GET")
+	r.Handle("/api/chef/nodes/{recipes}/{environment}/{groupid}", getChefNodes).Methods("GET")
```

Comment:
remove env and recipe from here since they contain special characters - only groupId